app/components/message_attachments/attachment_footer.tsx

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Text, View, Platform} from 'react-native';
+import FastImage from 'react-native-fast-image';
+
+import {Theme} from '@mm-redux/types/preferences';
+
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+type Props = {
+    text?: string;
+        icon?: string;
+        theme: Theme;
```

Comment:
indentation ?

Reply:
I expected the linter to find and fix that... :sweat_smile: